server/app/serializers/rpc/host_node_serializer.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../rpc_serializer'
+
+module Rpc
+  class HostNodeSerializer < RpcSerializer
+    attribute :id
+    attribute :created_at
+    attribute :updated_at
+    attribute :name
+    attribute :labels
+    attribute :overlay_ip
+    attribute :peer_ips
+    attribute :node_number
+    attribute :initial_member
+    attribute :grid
+
+    def id
+      object.node_id
+    end
+
+    def created_at
+      object.created_at.to_s
+    end
+
+    def updated_at
+      object.updated_at.to_s
+    end
+
+    def peer_ips
+      object.grid.host_nodes.ne(id: object.id).map{|n|
```

Comment:
As in https://github.com/kontena/kontena/pull/1906#discussion_r104626917, this logic is now duplicated in many places :(

It should be a `HostNode#peer_ips` method.